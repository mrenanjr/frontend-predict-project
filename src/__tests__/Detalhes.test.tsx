import * as React from 'react';
import {Link} from 'react-router-dom';
import { shallow } from 'enzyme';

const TestDetalhesValues = {
    location: {
        curso: 'Ciências da Computação',
        curso_abreaviado: 'cc',
        percent_evasao: 68,
        quant_aluno: 432,
        quant_evasao: 156
    }
}

test('Detalhes loads corretly', () => {
  const wrapper = shallow(<Link to={{ pathname: `/detalhes/cc`, state:  TestDetalhesValues }}/>);

  expect(wrapper).toMatchSnapshot();
});