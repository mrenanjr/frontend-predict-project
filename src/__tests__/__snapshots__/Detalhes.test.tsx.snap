// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detalhes loads corretly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
